packages/arb-token-bridge-ui/tests/e2e/specs/importToken.cy.ts

```diff
@@ -137,7 +137,7 @@ describe('Import token', () => {
         cy.findByPlaceholderText(/Search by token name/i)
           .should('be.visible')
           .type('UNI', { scrollBehavior: false })
-        cy.findByText('Uniswap').click({ scrollBehavior: false })
+        cy.findByTestId(/token-Uniswap/i).click({ scrollBehavior: false })
```

Comment:
Can we use scope instead and only look for token name in `TokenSearch` component?

Reply:
Done

Reply:
Used within() which does the same thing which you asked for - https://docs.cypress.io/api/commands/within